pkg/network/multus/nad.go

```diff
@@ -25,9 +25,10 @@ import (
 	"kubevirt.io/client-go/precond"
 )
 
-func GetNamespaceAndNetworkName(namespace string, fullNetworkName string) (string, string) {
+func GetNamespaceAndNetworkName(namespace, fullNetworkName string) (nadNamespace, nadName string) {
```

Comment:
Please avoid having named return values if its not a must.

Reply:
This change was per the linter's demand.
@EdDev do we want to relax the linter?

Reply:
No, we need to solve it by not returning two or more values of the same type.

Reply:
Either return a struct or create two types from strings.
Probably a struct is simpler.

Reply:
I would just relax the linter we know better what we want to enforce, no need to go hard on ourselves.
Returning multiple vars of the same times is not common in the codebase anyways.

Reply:
The function now returns a struct instead of two strings.